firefox-ios/Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -340,6 +340,8 @@ class TabManagerImplementation: LegacyTabManager, Notifiable {
 
             if featureFlags.isFeatureEnabled(.feltPrivacySimplifiedUI, checking: .buildOnly) {
                 store.dispatch(PrivateModeUserAction.setPrivateModeTo(tab.isPrivate))
+            } else {
+                store.dispatch(PrivateModeUserAction.setPrivateModeTo(false))
```

Comment:
Just to clarify: this will switch off `privateModeIsOn` within DefaultThemeManager, even if the current tab is a private tab, correct? Does that have any implications that will matter for our UI when the flag is disabled?

Reply:
It doesn't because the other features that the BVC state are gated behind the feature flag as well. So this essentially takes care of the theme (which is using the UserDefaults pref to know if it's on/off to overwrite the normal theme).

The "normal" experience - read, "pre Felt Privacy experience" - should be untouched and I didn't notice anything in my investigation and testing suggesting otherwise.